front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -43,6 +44,12 @@ export const stdcmConfSlice = createSlice({
     ) {
       state.maxSpeed = action.payload;
     },
+    updateTowedRollingStockID(
+      state: Draft<OsrdStdcmConfState>,
+      action: PayloadAction<OsrdStdcmConfState['towedRollingStockID']>
+    ) {
+      state.towedRollingStockID = action.payload;
+    },
     updateStdcmConfigWithData(
```

Comment:
There is something missing (that we also missed in the length/mass/vmax PR) : we should also pass here mass/length/vmax/towedRsId. Because if we switch between multiple simulation results, these fields aren't updated.

Reply:
The issue is still unsolved. The `selectedSimulation` doesn't get the new consist datas. They need to be add in `useStdcmForm`. (to test if it works, launch 2 different simulations and switch between them, the consist inputs should be automatically updated)

Reply:
Also when we change length/mass/vmax/towed, it doesn't display the simulation button again. This will probably be solved when you fix the above comment.